### PR TITLE
Change store job request  message structure.

### PR DESCRIPTION
**Pull request by prasanna08**
Changed store job controller request message structure according to [Oppia-ml Extension](https://github.com/oppia/oppia/wiki/Oppia-ml-Extension) page.

### CODE REVIEW

File: core/services/remote_access_services.py [hunk 1]
```diff
@@ -121,8 +121,10 @@ def store_trained_classifier_model(job_result_dict):
...
         raise Exception('job_result_dict must contain \'classifier_data\'.')
 
-    payload = job_result_dict
-    payload['vm_id'] = _get_vm_id()
+    payload = {
+        'message': job_result_dict,
+        'vm_id': _get_vm_id()
+    }
     signature = generate_signature(payload)
     payload['signature'] = signature
```
Comment (seanlip) [inline core/services/remote_access_services.py]: Hm. I thought we wanted the signature to be alongside the payload dict, not in it?

Comment (seanlip) [reply core/services/remote_access_services.py]: Oh, sorry, ignore me. I misread. But shouldn't the signature then be the signature of the message and not the payload?

Comment (pranavsid98) [reply core/services/remote_access_services.py]: @prasanna08 @seanlip Yep. We intended the signature to be generated with just the message.

Comment (prasanna08) [reply core/services/remote_access_services.py]: Yup forgot that.

Comment (prasanna08) [reply core/services/remote_access_services.py]: Done.

File: core/services/remote_access_services.py [hunk 2]
```diff
@@ -121,9 +121,11 @@ def store_trained_classifier_model(job_result_dict):
...
 
-    payload = job_result_dict
-    payload['vm_id'] = _get_vm_id()
-    signature = generate_signature(payload)
+    payload = {
+        'message': job_result_dict,
+        'vm_id': _get_vm_id()
+    }
+    signature = generate_signature(payload['message'])
     payload['signature'] = signature
```
Comment (anmolshkl) [inline core/services/remote_access_services.py:129]: @pranavsid98 @prasanna08 shouldn't we call this `message_signature` just to be explicit about what we mean by `signature`?

Comment (pranavsid98) [reply core/services/remote_access_services.py:129]: @anmolshkl We could do that. But then, signature is meant to be an encoding of the message we pass to the controller and the message is a key in the payload dict. It doesn't really seem to point to anything other than the message dict.

Comment (prasanna08) [reply core/services/remote_access_services.py:129]: @anmolshkl I don't have opinion on this but I don't see any need to make it explicit. We have already described the structure and what is what on wiki page, so it is not really necessary for the key to be explicit about it.

**Comment by prasanna08** [comment]
@seanlip, @anmolshkl can you pass this, if it is okay?

**Review by seanlip — APPROVED** [review]
LGTM

**Comment by prasanna08** [comment]
Thanks @seanlip

**Comment by seanlip** [comment]
No worries :) @anmolshkl, you're happy with this PR, right?

**Comment by anmolshkl** [comment]
@seanlip oh yes. Anyway, looks like it has been merged :)

**Pull request merged**